simd/src/CMakeLists.txt

```diff
@@ -19,6 +19,16 @@ install(
 # We have to pass the sources in here for Tribits
 # These will get ignored for standalone CMake and a true interface library made
 kokkos_add_library(kokkossimd SOURCES ${SIMD_SOURCES} HEADERS ${SIMD_HEADERS})
+
+if(KOKKOS_CXX_COMPILER_ID STREQUAL Intel OR KOKKOS_CXX_COMPILER_ID STREQUAL IntelLLVM)
+  find_library(INTEL_SVML NAMES svml)
+  if(INTEL_SVML)
+    message(STATUS "Found INTEL_SVML")
+    target_link_libraries(kokkossimd PUBLIC ${INTEL_SVML})
+    target_compile_definitions(kokkossimd PUBLIC KOKKOS_HAVE_INTEL_SVML)
```

Comment:
Why did you choose a compile definition over what we typically do, that is having it in the configured file?

Reply:
Introduced in https://github.com/kokkos/kokkos/pull/6581/commits/faa577d6f20d4ab0d3598497ebc6806dfa250f00. IIRC, I didn't want to create a new configuration file for SIMD and it didn't seem right to include it in `KokkosCore_config.h.in"`. A drawback is that we can't report it in `print_configuration`, though.